scripts/main.sh

```diff
@@ -53,13 +53,18 @@ do
 
     ## generate jobs -- TODO: customize names of the directory to avoid conflict if 2 tasks are started in 1 second
     echo "Generating run: (P=${C_PRIMARIES}, N=${N}, DIR=${RUN_DIR})"
-    #generatemc -p ${C_PRIMARIES} -j ${N} ${BASE_DIR}/input/data/ --workspace ${RUN_DIR} --scheduler_options "[--time=0:15:00 -A plgccbmc11-cpu]"
+    generatemc -p ${C_PRIMARIES} -j ${N} ${BASE_DIR}/input/data/ --workspace ${RUN_DIR} --scheduler_options "[--time=0:15:00 -A plgccbmc11-cpu]"
```

Comment:
Nie wiem czy nie trzeba będzie dać tu sleepa na sekundę, w związku z nazewnictwem stosowanym przez generatemc. Jakby nie patrzeć, jest to odpalane w pętli dla różnych wartości N